free-1.md

```diff
@@ -1,43 +1,53 @@
+
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This DSL allows users to get their food in the door when it comes to mixing music. Given how popular electronic music has become, I imagine that several fans probably have a few of their own novel ideas, but no outlet for them. I know personally that most existing audio mixing software is not only very pricey, but also extremely complicated. Simply learning how to use the basics requires several hour long tutorials on Youtube. This DSL plans to help amateurs who simply want some exposure to audio mixing without the time or capital investments.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+I think a DSL would be appropriate because, if designed effectively, audio mixing can be made more user-friendly. Most audio mixing software emulates a DJ console (the thing people use with a lot of knobs and dials and what not). Although it may seem counterintuitive, I felt audo mixing can be more effectively done through a language rather than a visual gui of a DJ console. To me, all the knobs and dials are pretty intimidating, and make it difficult to express ideas.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I once thought audio mixing would be an extremely easy thing to do. I had several ideas, but was met with more barriers to entry than I expected, including both the cost and complexity of existing software. Although these tools have been proven to be the best for the trade, these barriers alone were enough to make me abandon the idea. I felt like at the time there existed two extremely polarized types of audio mixing software: extremely powerful but extremely complicated, and extremely simple but extremely easy. I'm hoping this DSL will find somewhat of a middle ground. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+Music mixing for the amateur
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The program would take in any number of audio files and allow the user to apply some basic audio manipulation tools to it such as volume, bass, speed adjustments, cutting and looping tools, simultaneously playing multiple tracks, among others (mostly limited by the scope of the project). The user could combine some of these tools to create their own. Programming might look something like LateX in that it would look very structured with sequential blocks that correspond to parts of the music file (e.g. \begin{volume-boost}@3:44 ... \end{volume-boost}@3:55) or something along those lines. I think this is a correct way to approach the problem as it makes the structure of the music files more apparent without any visual ui. 
+
```

Comment:
Interesting! I would never have imagined something similar to LaTeX for this problem, but it could be a really fun way to approach mixing. As a LaTeX user, the format would appeal to me a lot. If you're trying to appeal to an amateur with limited technical skills, it might be worth thinking of a GUI or something that doesn't look quite as much like a programming language. I'd recommend interviewing a wide range of people.